From 616d7768375619ce04e17c10c44804cc6990251d Mon Sep 17 00:00:00 2001
From: Valentine Barshak <vbarshak@dev.rtsoft.ru>
Date: Thu, 24 Jan 2013 23:38:09 +0400
Subject: [PATCH 027/127] arm: tegra: ktt30: Enable SATA support in
 ktt30_defconfig

This enables Tegra SATA support in ktt30_defconfig

Signed-off-by: Valentine Barshak <vbarshak@dev.rtsoft.ru>
---
 arch/arm/configs/ktt30_defconfig |    4 +++-
 1 files changed, 3 insertions(+), 1 deletions(-)

diff --git a/arch/arm/configs/ktt30_defconfig b/arch/arm/configs/ktt30_defconfig
index 541aa26..6674b82 100644
--- a/arch/arm/configs/ktt30_defconfig
+++ b/arch/arm/configs/ktt30_defconfig
@@ -198,12 +198,14 @@ CONFIG_MPU_SENSORS_AK8975=m
 CONFIG_TEGRA_BB_SUPPORT=y
 CONFIG_TEGRA_BB_POWER=y
 CONFIG_TEGRA_BB_M7400=y
-CONFIG_SCSI=y
 CONFIG_BLK_DEV_SD=y
 CONFIG_BLK_DEV_SR=y
 CONFIG_BLK_DEV_SR_VENDOR=y
 CONFIG_CHR_DEV_SG=y
 CONFIG_SCSI_MULTI_LUN=y
+CONFIG_ATA=y
+CONFIG_SATA_AHCI_TEGRA=y
+# CONFIG_ATA_SFF is not set
 CONFIG_MD=y
 CONFIG_BLK_DEV_DM=y
 CONFIG_DM_CRYPT=y
-- 
1.7.4.4

